python/tk_multi_workfiles/file_save_form.py

```diff
@@ -445,9 +464,14 @@ def _on_browser_work_area_changed(self, entity, breadcrumbs):
         if entity:
             app = sgtk.platform.current_bundle()
             context = app.sgtk.context_from_entity_dictionary(entity)
-            env = WorkArea(context)
-        self._on_work_area_changed(env)
-        self._start_preview_update()
+
+            try:
+                env = WorkArea(context)
+            except TankError, e:
+                self._disable_save_and_warn(str(e))
```

Comment:
Is there an interesting place to put the whole stack trace?  Especially if debug logging is on?